zipline/finance/performance/position.py

```diff
@@ -172,13 +172,13 @@ def adjust_commission_cost_basis(self, asset, cost):
         if self.amount == 0:
             return
 
-        prev_cost = self.cost_basis * self.amount
+        prev_cost = self.cost_basis * abs(self.amount)
```

Comment:
After talking with @Jstauth and @abhijeetkalyan, treating cost basis as the price at which a position would break even is the correct call. Which means, the code is correct prior to this PR.

I'm going to take another pass to document that behavior and add test coverage for it.